test/modules/TestResultCoordinator/TestOperationResultStorage.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Storage;
+    using Microsoft.Azure.Devices.Edge.Storage.RocksDb;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class TestOperationResultStorage
+    {
+        static readonly ILogger Logger = ModuleUtil.CreateLogger("TestResultCoordinator");
+        static Dictionary<string, (string, ISequentialStore<TestOperationResult>)> resultStores;
+
```

Comment:
I think the storage should just save the TestOperationResult type based on the source, it should not care about result type

Reply:
Changed back